Sources/PrimerSDK/Classes/User Interface/Apple Pay/ApplePayViewModel.swift

```diff
@@ -160,19 +160,19 @@ class ApplePayViewModel: NSObject, ApplePayViewModelProtocol {
                                             router.presentErrorScreen(with: err)
                                         })
                                         
-                                        Primer.shared.delegate?.checkoutFailed(with: err)
+                                        Primer.shared.delegate?.checkoutFailed?(with: err)
                                     }
                                     
                                     completion(err)
                                     
                                 case .success(let token):
                                     DispatchQueue.main.async {
                                         if Primer.shared.flow.internalSessionFlow.vaulted {
-                                            Primer.shared.delegate?.tokenAddedToVault(token)
+                                            Primer.shared.delegate?.tokenAddedToVault?(token)
                                             completion(nil)
                                         } else {
                                             //settings.onTokenizeSuccess(token, completion)
-                                            Primer.shared.delegate?.authorizePayment(token, { (err) in
+                                            Primer.shared.delegate?.authorizePayment?(token, { (err) in
```

Comment:
[Q]
Is it possible/interesting to also write a log if authorizePayment is used?
Something like 
```
if (let authorizePayment = Primer.shared.delegate?.authorizePayment) {
  print("authorizePayment is deprecated, please use onTokenizeSuccess instead");
  authorizePayment(...)
}
```

Reply:
Yep, it is. Will do.